geekhours/database.py

```diff
@@ -94,6 +98,11 @@ def insert_donelist(self, date: str, course: str, duration: str):
         if check_duplicate:
             raise RuntimeError("Record already exists in 'donelist' table.")
 
+        if len(date) < 10:
```

Comment:
It would pass if the date is '12345678901' which means length is more than 10.

Reply:
Changed to  `if not len(date) == 10:`.